packages/typespec-azure-portal-core/test/decorators.test.ts

```diff
@@ -0,0 +1,172 @@
+import { ModelProperty } from "@typespec/compiler";
+import { BasicTestRunner, expectDiagnostics } from "@typespec/compiler/testing";
+import { deepEqual, strictEqual } from "assert";
+import {
+  getAboutDisplayName,
+  getAboutKeywords,
+  getAboutKeywordsItemsArray,
+  getAboutLearnMoreDocs,
+  getAboutLearnMoreDocsItemsArray,
+  getBrowseArgQuery,
+  getDisplayName,
+  getMarketplaceOfferId,
+} from "../src/decorators.js";
+import { createPortalCoreTestRunner } from "./test-host.js";
+
+describe("TypeSpec-Azure-Portal-Core decorators test", () => {
+  let runner: BasicTestRunner;
+
+  beforeEach(async () => {
+    runner = await createPortalCoreTestRunner();
+  });
+
+  it("test @displayName", async () => {
+    const { name } = await runner.compile(`
+        model Foo {
+            @test @displayName("nickName")
+            name: string;
+        }
+      `);
+    const actual = getDisplayName(runner.program, name as ModelProperty);
+    strictEqual(name.kind, "ModelProperty");
+    strictEqual(actual, "nickName");
+  });
+
+  // it("test @browse with file", async () => {
+  //   const browseOpt = {argQuery: {filePath:"./testfiles/query.kql"}} as BrowseOptions;
+  //   const { Foo } = await runner.compile(
+  //   `
+  //   @service({title: "Microsoft.Foo"})
+  //   @versioned(Versions)
+  //   @armProviderNamespace
+  //   namespace Microsoft.Foo;
+
+  //   @test @browse({argQuery:{filePath:"./testfiles/query.kql"}})
+  //   model Foo is TrackedResource<{}> {
+  //       @key("widgetName")
+  //       @segment("widgets")
+  //     @path
+  //       name: string;
+  //   }
+
+  //   model EmployeeProperties {
+  //     age?: int32 = 29;
+  //   }
+  //   `);
+  //   const result = getBrowseArgQuery(runner.program, Foo);
+  //   //const actual = getDisplayName(runner.program, name as ModelProperty);
+  //   strictEqual(Foo.kind, "Model");
+  // //strictEqual(actual, "nickName");
+  // });
+
+  it("test @browse with string", async () => {
+    const browseString = createbrowseTestDecorator(`"helloThisISArgQuery"`);
+    const { Foo } = await runner.compile(createTestSpec(browseString));
+    const result = getBrowseArgQuery(runner.program, Foo);
```

Comment:
I think it'll be more useful if the getBrowseArgQuery function returns a string rather than the model property where the string was declared, that way you will be able to just call it and get the value back rather than having to further process the ModelProperty node.